core/keyserver/revisions_test.go

```diff
@@ -148,8 +153,8 @@ func TestListMutations(t *testing.T) {
 		{desc: "exact page", pageSize: 6, start: 2, end: 7, wantNext: &rtpb.ReadToken{}},
 		{desc: "large page", pageSize: 10, start: 2, end: 7, wantNext: &rtpb.ReadToken{}},
 		{desc: "partial", pageSize: 4, start: 2, end: 6, wantNext: &rtpb.ReadToken{StartTime: &protopb.Timestamp{Nanos: 6}}},
-		{desc: "large page with token", token: MustEncodeToken(t, time.Unix(0, 3)), pageSize: 10, start: 3, end: 7, wantNext: &rtpb.ReadToken{}},
-		{desc: "small page with token", token: MustEncodeToken(t, time.Unix(0, 3)), pageSize: 2, start: 3, end: 5,
+		{desc: "large page with token", token: MustEncodeToken(t, time.Unix(0, 3*1000)), pageSize: 10, start: 3, end: 7, wantNext: &rtpb.ReadToken{}},
+		{desc: "small page with token", token: MustEncodeToken(t, time.Unix(0, 3*1000)), pageSize: 2, start: 3, end: 5,
 			wantNext: &rtpb.ReadToken{StartTime: &protopb.Timestamp{Nanos: 5}}},
```

Comment:
This Nanos here seems like a sign that the setup above is creating a world that cannot exist in the microsecond-granularity universe this PR is heading for.

Reply:
Indeed. Fixed.